applications/nrf5340_audio/README.rst

```diff
@@ -1268,6 +1270,10 @@ Legal notices for the nRF5340 Audio DK
 
       Nordic Semiconductor ASA customers using or selling these products for use in such applications do so at their own risk and agree to fully indemnify Nordic Semiconductor ASA for any damages resulting from such improper use or sale.
 
+   Radio frequency notice
+      The nRF5340 Audio development kit operates in the 2.4 GHz ISM radio frequency band.
+      The maximum radio-frequency power transmitted in the frequency band in which the development kit operates equals +3dBm (2 mW).
```

Comment:
no hyphen in "radio-frequency"?

Reply:
It looked like a modifier to me, but on second thought, I think indeed it is not needed.